int32                   id_index
int8                    object_count
UltrasonicTarget[16]    us_objects
